e2e/scenario_test.go

```diff
@@ -652,7 +652,33 @@ func Test_ubuntu2204AirGap(t *testing.T) {
 		},
 		Config: Config{
 			Cluster: ClusterKubenetAirgap,
-			VHD:     config.VHDUbuntu2204Gen2Containerd,
+			VHD:     config.VHDUbuntu2204Gen2ContainerdAirgapped,
+			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
+				nbc.ContainerService.Properties.AgentPoolProfiles[0].Distro = "aks-ubuntu-containerd-22.04-gen2"
+				nbc.AgentPoolProfile.Distro = "aks-ubuntu-containerd-22.04-gen2"
+
+				// TODO(xinhl): define below in the cluster config instead of mutate bootstrapConfig
```

Comment:
mutating bootstrap configuration is generally how we define scenarios, I'd avoid changing the base cluster config if possible

Reply:
remove the todo